workers/exclusive-content/README.md

```diff
@@ -0,0 +1,92 @@
+# Exclusive Content Worker
+
+This cloudflare worker will encrypt any plain text given a payment pointer. Furthermore, it will decrypt the resulting cypher text if it can verify payment by a receipt verifier.
+
+## Testing
+
+Configure your `wrangler.toml` file.
+
+```sh
+npm install
+wrangler secret put MASTERKEY
+# enter a key to be stored with cloudflare worker
+wrangler preview --watch
+```
+
+## Deploying
+
+Configure your `wrangler.toml` file.
+
+```sh
+npm install
+wrangler secret put MASTERKEY
+# enter a key to be stored with cloudflare worker
+wrangler publish
+```
+
+## Endpoints
+
+### `POST /encrypt`
+
+Encrypts plaintext
+
+#### Request Body
+
+JSON object containing
+| Name | Type | Description | Example |
+| -- | -- | -- | -- |
+| pp | String | (proxy) payment pointer | "\$spsp.example.com/alice" |
```

Comment:
we can get away with being more verbose than this; `plainText` and `paymentPointer` should be fine and will make life a little easier for us and any other consumers of this API

Reply:
Yes, I'll change it. It was just easier for all the testing because I didn't have to type that much ;-)